examples/applications/person_count_video.ipynb

```diff
@@ -0,0 +1,312 @@
+    "\n",
+    "        # Remove the assigned new box and person from further consideration\n",
+    "        ioma_matrix[i, :] = -1  # Invalidate this row (new box)\n",
+    "        # ioma_matrix[:, j] = -1  # Invalidate this column (person)\n",
+    "\n",
+    "    # Add any remaining unassigned new boxes as new persons\n",
+    "    for i, row in enumerate(ioma_matrix):\n",
+    "        if np.max(row) != -1:  # If this new box hasn't been assigned\n",
+    "            person_boxes.append([new_boxes[i]])\n",
+    "\n",
+    "    return person_boxes\n",
+    "\n",
+    "\n",
+    "def aggregate_person_boxes(res, threshold=0.6):\n",
+    "    # Extract and separate person, face, and head bounding boxes from results\n",
+    "    persons = [res.results[i] for i in range(len(res.results)) if res.results[i][\"label\"] == \"Person\"]\n",
+    "    heads = [res.results[i] for i in range(len(res.results)) if res.results[i][\"label\"] == \"Human head\"]\n",
+    "    faces = [res.results[i] for i in range(len(res.results)) if res.results[i][\"label\"] == \"Human face\"]\n",
+    "\n",
+    "    # Initialize person boxes with each face's box\n",
+    "    person_boxes = [[p] for p in faces]\n",
+    "\n",
+    "    # Add head and person detections to existing person detections\n",
+    "    person_boxes = add_detections_to_persons(person_boxes, heads, threshold)\n",
+    "    person_boxes = add_detections_to_persons(person_boxes, persons, threshold)\n",
+    "    return person_boxes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Analyzer\n",
+    "In order to annotate a video, a class known as an Analyzer is required. A child of this class will compute the number of people based on inference results, and annotate each frame with the computed value. This class, called PersonCounter, is implemented below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from numpy import ndarray\n",
+    "from degirum import _draw_primitives as dp\n",
+    "import itertools\n",
+    "\n",
+    "class PersonCounter(dgtools.ResultAnalyzerBase):\n",
+    "    \"\"\"Analyzer class to count people using inference results' bounding boxes.\"\"\"\n",
+    "    # Compute number of people in a given frame.\n",
+    "    def analyze(self, result):\n",
+    "        p_box = aggregate_person_boxes(result, threshold=threshold)\n",
+    "        result.person_count = len(p_box)\n",
+    "    \n",
+    "    # Annotate a video frame with the number of people in it.\n",
+    "    def annotate(self, result, image: ndarray) -> ndarray:\n",
+    "        prev_bbox = (0, 0, 0, 0)\n",
+    "        spacer = 3\n",
+    "\n",
+    "        draw = dp.create_draw_primitives(image, alpha=1.0, font_scale=1.0)\n",
+    "        current_color_set = itertools.cycle([(255, 255, 0)])\n",
+    "        overlay_color = next(current_color_set)\n",
```

Comment:
Hmm, strange code for color generation. Anyway, it will return (255,255,0), why to use itertools?
Proposition: either assign fixed color directly to `overlay_color` or deduce it from result:

overlay_color = result.overlay_color[0] if isinstance(result.overlay_color, list) else result.overlay_color

This way you will conform to model overlay_color selection.